## HG changeset patch
## User krh
## Date Mon Aug 15 06:25:05 2005
## Node ID 2bc9d793de28d2bb993dc195e80b4028a0fc4d16
## parent: b7042856813a5d4915ca53d1b78f64f39604a8d5
#cairo: changeset 2005-08-15 02:25:05 by krh
#
#- 2005-08-14 Kristian Høgsberg <krh@redhat.com>
#
#- src/cairo-ft-font.c (_cairo_ft_unscaled_font_lock_face): Zero out
#  'entry', not 'unscaled' when evicting face from cache.
#
#--- a/ChangeLog	Sun Aug 14 04:38:23 2005
#+++ b/ChangeLog	Mon Aug 15 06:25:05 2005
#@@ -1,3 +1,8 @@
#+2005-08-14  Kristian Høgsberg  <krh@redhat.com>
#+
#+	* src/cairo-ft-font.c (_cairo_ft_unscaled_font_lock_face): Zero
#+	out 'entry', not 'unscaled' when evicting face from cache.
#+
# 2005-08-13  Kristian Høgsberg  <krh@redhat.com>
# 
# 	* src/cairo-ps-surface.c: (_ps_output_fill_rectangles): Fix a
Index: src/cairo-ft-font.c
--- a/src/cairo-ft-font.c	Sun Aug 14 04:38:23 2005
+++ b/src/cairo-ft-font.c	Mon Aug 15 06:25:05 2005
@@ -504,8 +504,8 @@
 	    break;
 
 	FT_Done_Face (entry->face);
-	unscaled->face = NULL;
-	unscaled->have_scale = 0;
+	entry->face = NULL;
+	entry->have_scale = 0;
 
 	font_map->num_open_faces--;
     }

